src/test/resources/elastic-stack.yml

```diff
@@ -45,7 +45,7 @@ services:
       STATUS_ALLOWANONYMOUS: "true"
     image: docker.elastic.co/kibana/kibana:${ELASTIC_STACK_VERSION:-9.0.0}
     ports:
-      - "5601:5601"
+      - "5601"
```

Comment:
This change needs an additional change in the demo Docker compose file to expose a fixed port. 

https://github.com/jenkinsci/opentelemetry-plugin/blob/main/demos/docker-compose-elastic.yml#L57-L65

Reply:
Ok, reverted, please see https://github.com/jenkinsci/opentelemetry-plugin/pull/1145/commits/966487af4e8e040c4706682d56d88a7efbe59235